pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -253,27 +249,26 @@ def _close_logger(self, trainer) -> None:
         if trainer.logger is not None:
             trainer.logger.finalize("success")
 
-    @property
-    def xmp_spawn_kwargs(self):
+    def get_xmp_spawn_kwargs(self, trainer: "pl.Trainer") -> dict:
```

Comment:
something to address in the future: we are inheriting get_mp_spawn_args from the super class. 
I believe they should either be called the same or be made protected

Reply:
sounds good, will add a follow up PR!

Reply:
actually just fixed in this pr. please confirm if it looks good

Reply:
yes it looks nice, thank you